plugins/woocommerce-admin/webpack.config.js

```diff
@@ -133,7 +133,8 @@ const webpackConfig = {
 	},
 	plugins: [
 		...styleConfig.plugins,
-		new ForkTsCheckerWebpackPlugin(),
+		// Disable TS checker for now as it will block all development until all type inconsistencies are fixed
+		// new ForkTsCheckerWebpackPlugin(),
```

Comment:
Just dropping a reference to https://github.com/woocommerce/woocommerce/issues/32581. We should remember to re-enable it after TS errors are fixed